packages/batch-tx-handler/src/l1-batch-handler/utils.ts

```diff
@@ -165,18 +166,41 @@ export const getRawData = async (sequencerTx: string): Promise<[Uint8Array, BigN
   if (!tx || !txReceipt || (txReceipt && !txReceipt.status)) {
     throw new Error('No such a l1 transaction or transaction reverted');
   }
-
-  if (tx.to !== l2Network.ethBridge.sequencerInbox) {
+  if (tx.to.toLowerCase() !== l2Network.ethBridge.sequencerInbox.toLowerCase()) {
     throw new Error('Not a sequencer inbox transaction');
   }
 
   const funcData = contractInterface.decodeFunctionData('addSequencerL2BatchFromOrigin', tx.data);
   const seqData = funcData['data'].substring(2); //remove '0x'
   const deleyedCount = funcData['afterDelayedMessagesRead'] as BigNumber;
-  const rawData = Uint8Array.from(Buffer.from(seqData, 'hex'));
+  let rawData = Uint8Array.from(Buffer.from(seqData, 'hex'));
+  if (rawData[0] & DASMessageHeaderFlag) {
+    if (l2Network.chainID !== 42170) {
+      throw new Error('For anytrust network, only support Arbitrum nova now');
+    }
+    rawData = await processDASBatch(rawData);
+  }
   return [rawData, deleyedCount];
 };
 
+const processDASBatch = async (rawData: Uint8Array) => {
+  const req = await fetch(NovaDacUrl);
+  const urls = (await req.text()).split('\n');
+  if (urls.length === 0) {
+    throw Error('No online das servers now');
+  }
+  return getDACData(urls[0], rawData);
+};
+
+const getDACData = async (url: string, rawData: Uint8Array) => {
+  // The first byte is header flag, the 2nd to 33rd bytes is keyset hash, 34th to 65th is data hash which is what we want.
```

Comment:
Can you add a reference URL here? (the nitro code that creates this concatenated info for example)
If this changes in the future (or we want to support different formats coming from other DAS for example), it might be good to have the reference here